app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt

```diff
@@ -104,6 +104,12 @@ class ProjectAdapter(private val context: Context, private val utils: Utils, pri
                 holder.chipGroupView.addView(chipView)
             }
 
+            for(section in project.allowedSections){
+                val chipView: Chip = Chip(context)
+                chipView.text = section
+                holder.chipGroupView.addView(chipView)
+            }
```

Comment:
The tags and sections can't be differentiated. Maybe separating them or changing color of one type could be better,